packages/core-components/src/components/Select/Select.tsx

```diff
@@ -126,12 +122,17 @@ export type SelectItem = {
 export type SelectedItems = string | string[] | number | number[];
 
 export type SelectProps = {
+  className?: string;
   multiple?: boolean;
   items: SelectItem[];
+  showLabel?: boolean;
```

Comment:
want to open discussion about this solution here.

It's a hack, I see you use label as kinda key, so I can't make it optional without some breaking refactoring. What can you suggest here ?